stdlib/database/mysql/src/main/ballerina/mysql/endpoint.bal

```diff
@@ -26,7 +26,7 @@ import ballerina/sql;
 # + poolOptions - Properties for the connection pool configuration. Refer `sql:PoolOptions` for more details
 # + dbOptions - A map of DB specific properties
 public type ClientEndpointConfig record {
-    string host = "";
+    string host = "localhost";
```

Comment:
Shouldn't we mandate this parameter by having it as "";

Reply:
Empty string still won't make it mandatory. To make it a "true" required field, you shouldn't provide any explicit default values.

Reply:
But this can leads to mistakes in the environments. For ex users can get connect to localhost mistakenly even if they want to connect to production environments.  So this should be a mandatory parameter